src/sagemaker/automl/automl.py

```diff
@@ -187,49 +248,29 @@ def list_candidates(
 
         return self.sagemaker_session.list_candidates(**list_candidates_args)["Candidates"]
 
-    def deploy(
+    def create_model(
```

Comment:
Should this actually be private? I don't see a way to deploy an already created model here, maybe it exists outside of automl.py?

Reply:
no. I purposefully introduced the method create_model as public facing APIs. This does not really create the sagemaker model, but it allows to use the generated PipelineModel to perform batch transform as well.

see the test for an example usecase